debian/jitsi-meet-web-config.postinst

```diff
@@ -208,12 +220,30 @@ case "$1" in
             invoke-rc.d apache2 reload || true
         fi
 
-        echo "----------------"
-        echo ""
-        echo "You can now switch to a Let’s Encrypt certificate. To do so, execute:"
-        echo "/usr/share/jitsi-meet/scripts/install-letsencrypt-cert.sh"
-        echo ""
-        echo "----------------"
+        if [ "$ISSUE_LE_CERT" = "true" ] ; then
+            export HOME=/opt/acmesh
+            curl https://get.acme.sh | sh -s email=$EMAIL
```

Comment:
What about bundling acme.sh in resources? Not a fan of downloading it like this...

Reply:
That installs itself on the machine and can be later upgraded, if we add one it will be forever old ... I don't know ... I would love to see it in debian package dependency.

Reply:
FWIW, on the Docker setup I haven't updated the version ever since I introduced it: https://github.com/jitsi/docker-jitsi-meet/blob/47804d0b9e1657debf0200472e3eb22563a26100/web/Dockerfile#L11

They support many features we don't really care about, so I've taken the conservative route and it seems to work ok. It will likely need an update, or maybe we could attempt an update when invoking it...